components/media-feature/Component.jsx

```diff
@@ -111,8 +112,8 @@ function MediaType({ type }) {
 function FeaturedHeading({ type }) {
   switch (type) {
     case "Podcast":
-      return <SidebarHeading icon="Featured audio" title="Featured audio" />;
```

Comment:
This should be reverted. The text label next to the icon no longer prints as shown in the designs.

Reply:
Done, but I changed <SidebarHeading>. See below.